src/components/SingleClinic.css

```diff
@@ -0,0 +1,76 @@
+.Addfield {
+  display: flex;
+  justify-content: center;
+}
+
+.SingleClinicView {
+  background-color: #ffffff;
+  box-shadow: 0px 0px 5px 8px rgba(226, 221, 221, 0.3);
+  width: 80%;
+  border-radius: 15px;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 60px;
+  padding-bottom: 40px;
+  margin-bottom: 90px;
+  margin-top: 40px;
+  height: auto;
+  justify-content: space-around;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+}
+
+.InputPlace {
+  box-shadow: 8px 8px 8px #cbced1, inset -8px -8px 8px #e2ecf8;
+  padding-bottom: 40px !important;
```

Comment:
I believe that using !important is bad practice. Not to sure. Probably there for a reason:)